CHANGES.rst

```diff
@@ -110,7 +116,7 @@ Unreleased
 
 **Expected Changes**
 
-* The behavior of :class:`.APIException` will no longer unicode-escape strings
+* The behavior of :class:`.RedditAPIException` will no longer unicode-escape strings
```

Comment:
Let's avoid updating old error log entries, other than to fix errors from sphinx not being able to link when something has been deleted.

Reply:
This is why I did the update, aliases aren't recognized as classes.

Reply:
I'd suggest making them unlinkable in that case by double backticks.